api/v1alpha1/serviceresolver_types.go

```diff
@@ -306,6 +306,11 @@ func (in *ServiceResolver) Validate(namespacesEnabled bool) error {
 	return nil
 }
 
+// DefaultNamespaceFields has no behaviour here as service-resolver have namespace fields
+// that do not default.
+func (in *ServiceResolver) DefaultNamespaceFields(_ bool, _ string, _ bool, _ string) {
+}
```

Comment:
I'm curious why we didn't need to say `return` here as we did in others.

Reply:
it was unintentional. Fixed.